doc/conf.py

```diff
@@ -193,7 +193,7 @@
         "version_match": version,
     },
     "navbar_start": ["navbar-logo", "version-switcher"],
-    "announcement": "HyperSpy API is changing in version 2.0, see <a href='https://hyperspy.org/hyperspy-doc/dev/changes.html'>upcoming changes!</a>",
+    "announcement": "HyperSpy API is changing in version 2.0, see the <a href='https://hyperspy.org/hyperspy-doc/current/changes.html'>release notes!</a>",
```

Comment:
This will work when the documentation is pushed to https://github.com/hyperspy/hyperspy-doc.